docs/source/fitting/fitfunctions/LatticeFunction.rst

```diff
@@ -50,19 +50,18 @@ used to generate Bragg reflections that are expected for the crystal structure o
 
     a_true = 5.4311946
     a = np.round(parameters.cell(0, 1), 7)
-    a_err = np.round(parameters.cell(0, 2), 16)
+    a_err = np.round(parameters.cell(0, 2), 7)
 
     print("Refined lattice parameter: a = {0} +/- {1}".format(a, a_err))
     print("Difference from expected value: a_observed - a_expected = {0}".format(np.round(a - a_true, 7)))
-    print("Is this difference within the standard deviation? {0}".format("Yes" if np.fabs(a - a_true) <= a_err else "No"))
```

Comment:
Is there no way to get this to match?

Reply:
There is no noise in the fitted data which could give the standard deviation one or two solid significant digits. We could manually set the limit on it about 1e-15 - 1e-14 which is the accuracy of double precision calculations. But I don't think it is useful.